log/development.log

```diff
@@ -2492,6 +2492,136 @@ Completed 200 OK in 37ms (Views: 35.9ms | ActiveRecord: 0.3ms)
 
 Started GET "/assets/application.self-e80e8f2318043e8af94dddc2adad5a4f09739a8ebb323b3ab31cd71d45fd9113.css?body=1" for ::1 at 2016-01-04 17:14:22 -0500
 
+<<<<<<< HEAD
```

Comment:
We added it to our .gitignore file, but .git somehow seems to still be tracking it. We plan to copy the contents of the file, delete, recreate, and paste...